frontend/src/layout/WorkerStats.js

```diff
@@ -26,12 +26,14 @@ export function WorkerStats() {
     }, [])
 
     return heartbeat !== null ? (
-        <span style={{ marginRight: 32, marginLeft: -16 }}>
+        <span>
             {heartbeat === 'offline' || heartbeat === 'error' || heartbeat > 90 ? (
```

Comment:
If you're touching this file could you do me a massive favour and hide this if `window.location.href.indexOf('app.posthog.com') > -1` ?

Reply:
done